src/main/java/io/github/apace100/origins/origin/Origin.java

```diff
@@ -33,13 +34,14 @@ public class Origin {
         .add("impact", OriginsDataTypes.IMPACT, Impact.NONE)
         .add("loading_priority", SerializableDataTypes.INT, 0)
         .add("upgrades", OriginsDataTypes.UPGRADES, null)
+        .add("textures", ApoliDataTypes.IDENTIFIER_MAP, new LinkedHashMap<>())
```

Comment:
I think the name of the field should be `texture_map`, just to be consistent with the `texture_map` field of the `status_bar_texture` power type that behaves the same

Reply:
oh yeah for sure, i forgot to check the name of the existing one, will change when i'm able to